spec/datadog/appsec/waf_spec.rb

```diff
@@ -2191,7 +2215,7 @@
       end
 
       let(:matching_input_status) do
-        { 'server.response.status' => 404 }
+        { 'server.response.status' => '404' }
```

Comment:
This change is also concern to the upgrade with  1.14.0

> In order to ensure that the extracted schema is accurate, tracers must provide the original scalar types to the WAF, rather than just strings, during the rule evaluation stage (ddwaf_run). The rules have been checked to ensure that this will not cause any issues.
This applies to all addresses EXCEPT server.response.status, which needs to be provided as a string. Rules are currently expecting the response code as a string and, since the WAF doesn’t perform type conversions during rule evaluation, providing it as an integer would prevent relevant rules from being evaluated.